content/docs/csidriver/release/powerflex.md

```diff
@@ -13,25 +13,17 @@ description: Release notes for PowerFlex CSI driver
 
 
 
+
 ### New Features/Changes
 
-- [#1359 - [FEATURE]: Add Support for OpenShift Container Platform (OCP) 4.16 ](https://github.com/dell/csm/issues/1359)
-- [#1400 - [FEATURE]: Support for Kubernetes 1.30](https://github.com/dell/csm/issues/1400)
-- [#1358 - [FEATURE]: Support for PowerFlex 4.6](https://github.com/dell/csm/issues/1358)
-- [#1397 - [FEATURE]: Observability upgrade is supported in CSM Operator](https://github.com/dell/csm/issues/1397)
+- [#1472 - [FEATURE]: Support for Kubernetes 1.31](https://github.com/dell/csm/issues/1472)
+- [#1508 - [FEATURE]:  Add Support for Kubevirt](https://github.com/dell/csm/issues/1508)
```

Comment:
This should be KubeVirt not "Kubevirt" I updated the GitHub Issue. https://github.com/dell/csm/issues/1508